web/ui/src/app/lineage/graph/graph.model.ts

```diff
@@ -70,11 +78,12 @@ export class RegularVisNode extends VisNode {
 }
 
 export class HighlightedVisNode extends VisNode {
-    constructor(public operation: IOperation) {
+    constructor(
+        public operation: IOperation, public label : string) {
```

Comment:
Minor thing: Line breaks. For a multi-line method/constructor signature, it's a good practice to put every argument on its own line. So you either place everything on one line or one arg per line.

Reply:
corrected